docs/instrumentations/golang/golang.mdx

```diff
@@ -19,28 +19,35 @@ The following go modules will be auto instrumented by Odigos:
 
 ### HTTP Frameworks
 
-- [`github.com/gin-gonic/gin`](https://pkg.go.dev/github.com/gin-gonic/gin) <sup>*</sup>. HTTP web framework
 - [`net/http`](https://pkg.go.dev/net/http) go standard library for http server and client
 
+<Warning>Please note that in order for the `http.route` parameter to be properly extracted by Go auto instrumentation, 
+you should use Go 1.22 or above and declare a template handler as described in [this article](https://go.dev/blog/routing-enhancements).</Warning>
+
+- [`github.com/gin-gonic/gin`](https://pkg.go.dev/github.com/gin-gonic/gin) HTTP web framework ⭐️ 
+
 ### HTTP Clients
 
 - [`net/http`](https://pkg.go.dev/net/http) go standard library for http server and client
+<Warning>Please note that in order for the `http.route` parameter to be properly extracted by Go auto instrumentation, 
+you should use Go 1.22 or above and declare a template handler as described in [this article](https://go.dev/blog/routing-enhancements).</Warning>
```

Comment:
This should be added only for the server (not the client)

Reply:
oh, thanks for the heads up, updating it